techtv2ovs/tasks.py

```diff
@@ -0,0 +1,90 @@
+""" Celery tasks for techtv2ovs """
+
+from os.path import splitext
+
+import boto3
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+
+from techtv2ovs.constants import TTV_VIDEO_BUCKET, ImportStatus
+from techtv2ovs.models import TechTVVideo
+from ui.models import VideoFile
+
+log = get_task_logger(__name__)
+
+# pylint: disable=unused-argument,broad-except
+
+
+def parse_encoding(filename):
+    """
+    Get the correct encoding base on the video file's name
+
+    Args:
+        filename (str): The video file name
+
+    Returns:
+        str: The encoding for the videofile.
+
+    """
+    basename = splitext(filename)[0].split('/')[-1]
+    return basename.upper() if basename == 'hd' else basename
+
+
+@shared_task(bind=True)
+def process_videofiles(self, ttv_id, files, run_copy):
+    """
+    Copy over S3 video files for a TechTV video, and create a VideoFile object for each.
+    Update the TechTVVideo and Video statuses when done (complete, error, or missing).
+
+    Args:
+        ttv_id (int): The ID of the TechTVVideo object
+        files (list): A list of S3 videofile objects
+        run_copy (bool): Copy S3 files between buckets
+    """
+    ttv_video = TechTVVideo.objects.get(id=ttv_id)
+    ttv_video.videofile_status = ImportStatus.CREATED
+    for file in files:
+        try:
+            # Copy S3 objects to OVS buckets (if not already there)
+            src = {'Bucket': TTV_VIDEO_BUCKET, 'Key': file['Key']}
+            if 'original' in file['Key']:
+                dst = {
```

Comment:
It looks like we are not ever using `dst` itself, just the bucket and the key inside it. Can you replace with separate variables for the both of them?

Reply:
✅